pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -177,10 +184,12 @@ func (m *kubeGenericRuntimeManager) configureContainerSwapResources(lcr *runtime
 	// NOTE(ehashman): Behavior is defined in the opencontainers runtime spec:
 	// https://github.com/opencontainers/runtime-spec/blob/1c3f411f041711bbeecf35ff7e93461ea6789220/config-linux.md#memory
 	switch m.memorySwapBehavior {
+	case kubelettypes.NoSwap:
+		swapConfigurationHelper.ConfigureNoSwap(lcr)
 	case kubelettypes.LimitedSwap:
 		swapConfigurationHelper.ConfigureLimitedSwap(lcr, pod, container)
 	default:
-		swapConfigurationHelper.ConfigureUnlimitedSwap(lcr)
+		swapConfigurationHelper.ConfigureNoSwap(lcr)
```

Comment:
hm can we do this to a beta feature? We're taking a former default with a beta feature gate and then totally upending the meaning.

Reply:
Yea :( The default we had to drop and that is why I documented in the KEP. It is a breaking change but we want to drop that support so not sure what we should do in that case.

Reply:
https://github.com/kubernetes/enhancements/blob/e14ee585ad86ae38dd13583bedee2d25f9eadaf1/keps/sig-node/2400-node-swap/README.md#beta-2
gotcha missed that piece